### ISSUE TITLE
Recurring transactions

### ISSUE THREAD

**Issue by Arcesilas** [Winbee/cashcash-desktop#13]
Do you plan adding support for recurring transactions?

We should be able to choose the periodicity : every n days/weeks/months where 1 < n < 30 (days), 1 < n < 4 (weeks), 1 < n < 12 (months).
We should also be able to allow prevent transaction to occure during week-end and choose if the transaction should be set before (friday), after (monday) or befor/after (friday instead of saturday and monday instead of sunday).

**Comment by Winbee** [comment]
Good idea @Arcesilas. Yeah, it would probably be not too big to implement but I have no idea how to do it :-/.
